app/assets/javascripts/utilities/sendFetch.js

```diff
@@ -30,7 +30,7 @@ function sendFetch(switchStatement, body) {
       });
     case 'reaction-creation':
       return fetchCallback({
-        url: '/reactions',
+        url: '/add_articles',
```

Comment:
Again, great use of previous code and pattern matching to refactor new functionality.